pyHS100/pyHS100.py

```diff
@@ -37,37 +37,7 @@ class SmartPlugException(Exception):
     """
     pass
 
-
-class SmartPlug(object):
-    """Representation of a TP-Link Smart Switch.
-
-    Usage example when used as library:
-    p = SmartPlug("192.168.1.105")
-    # print the devices alias
-    print(p.alias)
-    # change state of plug
-    p.state = "ON"
-    p.state = "OFF"
-    # query and print current state of plug
-    print(p.state)
-
-    Errors reported by the device are raised as SmartPlugExceptions,
-    and should be handled by the user of the library.
-
-    Note:
-    The library references the same structure as defined for the D-Link Switch
-    """
-    # switch states
-    SWITCH_STATE_ON = 'ON'
-    SWITCH_STATE_OFF = 'OFF'
-    SWITCH_STATE_UNKNOWN = 'UNKNOWN'
-
-    # possible device features
-    FEATURE_ENERGY_METER = 'ENE'
-    FEATURE_TIMER = 'TIM'
-
-    ALL_FEATURES = (FEATURE_ENERGY_METER, FEATURE_TIMER)
-
+class SmartDevice(object):
```

Comment:
I can't figure out what this complaint is - there appear to be two blank lines after this patch.